### PR TITLE
add function to retrieve the master key for secret manager

### PR DESCRIPTION
**Pull request by wanghouqi**
if encryptionKeyId is not provided, then use the default one as the master key for secret manager.

test: passed the python test

Signed-off-by: wanghouqi <wanghouqi@vip.qq.com>

### CODE REVIEW

File: ehsm_kms_service/secret_manager_apis.js [hunk 1]
```diff
@@ -165,24 +192,12 @@ const createSecret = async (res, appid, payload, DB) => {
...
-            }
-        } else if (encryptionKeyId == "" || encryptionKeyId == undefined) {
-            let default_cmk_res = await DB.partitionedFind('user_info', queryDefaultCMK(appid))
-            if (default_cmk_res.docs.length == 0) {
-                res.send(_result(400, 'Cannot find default CMK'))
-                return
-            }
-            keyBlob = default_cmk_res.docs[0].sm_default_cmk
+        // query cmk and encrypt secretData
+        retCMK = await queryCMK(DB, appid, encryptionKeyId)
```
Comment (syan10) [inline ehsm_kms_service/secret_manager_apis.js:196]: change the function name as get_SM_Masterkey

and better to change the keyBlob -> SM_MasterKey

Comment (wanghouqi) [reply ehsm_kms_service/secret_manager_apis.js:196]: It will be change

File: ehsm_kms_service/secret_manager_apis.js [hunk 2]
```diff
@@ -64,27 +64,54 @@ function check_rotationInterval_format(rotationInterval) {
...
-
-//Defaultcmk query
-const queryDefaultCMK = (appid) => {
-    return {
-        selector: {
-            appid
-        },
-        fields: ['sm_default_cmk'],
-        limit: 1
+    if (encryptionKeyId != undefined && encryptionKeyId != '' && typeof (encryptionKeyId) == 'string') {
```
Comment (yang8621) [inline ehsm_kms_service/secret_manager_apis.js:74]: I'd like to exchange Line74{} and Line95{} as:
if (encryptionKeyId == "" || encryptionKeyId == undefined) {
......
}
else if (typeof (encryptionKeyId) == 'string') {
.......
}
else {
......
}

Comment (wanghouqi) [reply ehsm_kms_service/secret_manager_apis.js]: The if logic will be change

**Review by yang8621 — APPROVED** [review]

**Review by syan10 — APPROVED** [review]

**Pull request merged**